satpy/etc/composites/abi.yaml

```diff
@@ -215,3 +215,267 @@ composites:
+    prerequisites:
+    - name: C07
+    - name: C14
+    - name: C15
+    standard_name: night_microphysics
+
+  convection:
+    compositor: !!python/name:satpy.composites.Convection
+    prerequisites:
+    - name: C02
+    - name: C05
+    - name: C07
+    - name: C08
+    - name: C10
+    - name: C14
+    standard_name: convection
+
+## Cloud Phase Distinction or Day Cloud Phase (name from NASA SPORT) or Cloud Type RGB(name from EUMETSAT)
+  cloud_phase_distinction_raw:
+    compositor: !!python/name:satpy.composites.SelfSharpenedRGB
+    prerequisites:
+    - name: C13
+    - name: C02
+    - name: C05
+    standard_name: cloud_phase_distinction
+
+  cloud_phase_distinction:
+    compositor: !!python/name:satpy.composites.SelfSharpenedRGB
+    prerequisites:
+    - name: C13
+    - name: C02
+      modifiers: [sunz_corrected, rayleigh_corrected]
+    - name: C05
+      modifiers: [sunz_corrected]
+    standard_name: cloud_phase_distinction
+
+## Water Vapors RGB or Simple Water Vapor (name from NASA SPORT)
+  water_vapors1:
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: C13
+    - name: C08
+    - name: C10
+    standard_name: water_vapors1
+
+  mid_vapor:
+    compositor: !!python/name:satpy.composites.DifferenceCompositor
+    prerequisites:
+    - name: C10
+    - name: C08
+    standard_name: mid_vapor
+## Water Vapors RGB (other version) or Differential Water Vapor (name from NASA SPORT)
+  water_vapors2:
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: mid_vapor
+    - name: C10
+    - name: C08
+    standard_name: water_vapors2
+
+## Day Cloud Phase RGB(name from EUMETSAT)
+  cloud_phase_raw:
+    compositor: !!python/name:satpy.composites.SelfSharpenedRGB
+    prerequisites:
+    - name: C05
+    - name: C06
+    - name: C02
+    standard_name: natural
+
+  cloud_phase:
+    compositor: !!python/name:satpy.composites.SelfSharpenedRGB
+    prerequisites:
+    - name: C05
+      modifiers: [sunz_corrected]
+    - name: C06
+      modifiers: [sunz_corrected]
+    - name: C02
+      modifiers: [sunz_corrected, rayleigh_corrected]
+    standard_name: natural
+##
+  ir_overview:
+    compositor: !!python/name:satpy.composites.SelfSharpenedRGB
+    prerequisites:
+    - name: C05
+      modifiers: [co2_corrected]
+    - name: C14
+    - name: C15
+    standard_name: ir_overview
+## Airmass using Channel14 RGB (own)
+  airmass_C14:
+    compositor: !!python/name:satpy.composites.Airmass
+    prerequisites:
+    - name: C08
+    - name: C10
+    - name: C12
+    - name: C14
+    standard_name: airmass
+## Tropical Airmass RGB (name from EUMETSAT)
+  airmass_tropical:
+    compositor: !!python/name:satpy.composites.TropicalAirmass
+    prerequisites:
+    - name: C08
+    - name: C10
+    - name: C12
+    - name: C13
+    standard_name: airmass_tropical
+## Ash, Dust and fog using RGB Dust recipe
+  ash_abi:
```

Comment:
These should just be called `ash` since they are in `abi.yaml` now and I'm guessing the `ash` recipe in `visir.yaml` doesn't work for ABI.